nemo/collections/nlp/modules/common/text_generation_strategy.py

```diff
@@ -487,6 +544,7 @@ def __init__(self, model):
             is_multimodal=self.data_cfg.is_multimodal,
             sep_image_conv_front=self.data_cfg.sep_image_conv_front,
             conv_template=self.data_cfg.get("conv_template", "nvgpt"),
+            model_type=self.cfg.mm_cfg.llm.get("model_type", "nvgpt"),
```

Comment:
should we rename this to nemotron?

Reply:
that will break many old checkpoint, we might just keep this way for a while...